envFiles/.env.devcontainer

```
@@ -142,10 +140,34 @@ REDIS_TEST_MAPPED_PORT=6378
+API_OTEL_ENABLED=true
+API_OTEL_ENVIRONMENT=local
+# NOTE: Set to your OTLP collector endpoint. Use localhost:4318 if running Jaeger locally,
+# or add a 'jaeger' service to docker/compose.devcontainer.yaml if using container networking.
+API_OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318/v1/traces
+API_OTEL_SAMPLING_RATIO=1.0
+API_OTEL_SERVICE_NAME=talawa-api
+
+API_SLOW_REQUEST_MS=500
+
+##### Email Configuration #####
+<<<<<<< HEAD
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The <<<<<<< HEAD key should be with a value or have an equal sign

(KeyWithoutValue)